.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -157,15 +185,128 @@ jobs:
         cmake --build $GITHUB_WORKSPACE/build --target utils/llvm-lit/install
         cmake --build $GITHUB_WORKSPACE/build --target install-clang-format
         cmake --build $GITHUB_WORKSPACE/build --target install-clang-tidy
+        cmake --build $GITHUB_WORKSPACE/build --target install-llvm-size
+        # TODO this should be resolved in CMakeLists.txt
+        cmake --build $GITHUB_WORKSPACE/build --target install-lld || echo "skipped"
 
-    - name: Pack
+    - name: Pack toolchain
       run: tar -cJf llvm_sycl.tar.xz -C $GITHUB_WORKSPACE/build/install .
-    - name: Upload artifacts
+    - name: Pack LIT
+      run: tar -cJf lit.tar.xz -C $GITHUB_WORKSPACE/src/llvm/utils/lit .
```

Comment:
LIT can be installed with pip - https://llvm.org/docs/TestSuiteGuide.html#quickstart.
Would it be more optimal to install it into docker images used for testing?

Reply:
From what I see, these are only released versions of lit. Although, it does not get updates that often, I'm still not sure if there's any kind of compatibility guarantees for tests and lit.